CHANGELOG_UNRELEASED.md

```diff
@@ -101,7 +102,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Replaced the legacy generalised induction idiom with a more robust one
 that does not rely on the `{-2}` numerical occurrence selector, using
 new `ssrnat` helper lemmas `ltn_ind`, `ubnP`, `ubnPleq`,  ...., (see above). The new idiom is documented in `ssrnat`.
-   This change anticipates an expected evolution of `fintype` to integrate `finmap`. It is likely that the new definition of the `#|A|` notation will hide multiple occurrences of `A`, which will break the `{-2}` induction idiom. Client libraries should update before the 1.11 release.
+   This change anticipates an expected evolution of `fintype` to integrate `finmap`. It is likely that the new definition of the `#|A|` notation will hide multiple occurrences of `A`, which will break the `{-2}` induction idiom. Client libraries should update before the 1.11 release (see PR #434 for examples).
+   
+ - Replaced the use of the accidental convertibility between `enum A` and 
+   `filter A (index_enum T)` with more explicit lemmas `big_enumP`, big_enum`, `big_enum_cond`, `big_image` added to the `bigop` library. This convertibilty, and indeed equality may no longer hold in future `mathcomp` releases when sets over `finType`s are generalised to finite sets over `choiceType`s, so client libraraies should stop relying on this convertibility, as well as the `filter_index_enum` lemma. File `bigop.v` has some boilerplate to help with the port; also see PR #441 for examples.
+   
+ - Restricted `big_image`, big_image_cond`, `big_image_id` and `big_image_cond_id`
+ to `bigop`s over _abelian_ monoids, anticipating the change in the definition of `enum`. This may introduce some incompatibilities - non-abelian instances should be dealt with a combination of `big_map` and `big_enumP`.
```

Comment:
Is there an example of this in the library right now, if so can you indicate the theorem + file?

Reply:
There isn't, and indeed none of the `big_image` lemmas were used in the library at any point, and almost all the instances where `big_map` + `big_filter` or `enumT` were used as a proxy for `big_image` _were_ in the abelian case, so the PR actually introduces `big_image` there. The only exception is the `split_diagA` technical lemma in `mxpoly`, but it's non-standard because I actually _define_ `diagA` to be the `map` over the enumeration that is meant to be abstracted by `big_enumP`, which means I don't use `big_enumP` to abstract over it right after I use `big_map`, as I normally should. That twist is due to the fact I can't use a lemma like `big_enumP` to create constants at the vernacular level.